universe/space/space_attributes.go

```diff
@@ -234,6 +238,10 @@ func (s *Space) UpdateSpaceAttributeValue(
 		}
 	}()
 
+	if s.world != nil {
+		s.world.GetCalendar().OnAttributeUpsert(attributeID, payload.Value)
+	}
```

Comment:
same
https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/space/space_attributes.go#L395

Reply:
hooked registered in new place